package.json

```diff
@@ -34,7 +34,8 @@
     "prepare": "npm run build",
     "start": "microbundle-crl watch --no-compress --format modern,cjs",
     "storybook": "start-storybook -p 9009",
-    "test": "jest",
+    "test": "cross-env NODE_ICU_DATA=node_modules/full-icu jest",
```

Comment:
What is full-icu?

Reply:
I read the README for it and I still don't understand what it does. Something about Unicode?

Reply:
Yes, for example, on windows machine, Intl.NumberFormat('de-DE', { style: 'currency', currency: 'EUR' }) is going to return in string close to '€ 20,000.00' while in others, such as linux (assuming) would be '20.000,00 €'.  Tests were failing in circleCI because of this.

Reply:
Not sure I fully understand the underlying reason, but I get that there could be platform differences in rendering text. Thanks for the explanation, let's keep it.